### DISCUSSION TITLE
Acatama for multitemporal validación

### DISCUSSION THREAD

**Discussion by efrainturpo** [SMByC/AcATaMa#16] [Ideas]
Currently, satellite image classification works are carried out at a multitemporal level. I think it would be interesting to have that option in the plugin, an example so that 30 multitemporal maps generated from landsat can be validated in GEE.

**Comment by XavierCLL** [comment]
Hi @efrainturpo 

Thanks, but I am not sure what you mean exactly; validate more than one thematic map at the same time? or having a multitemporal images as reference data for labeling? (the latter is currently possible in Acatama)

**Comment by efrainturpo** [comment]
Thank you very much for the reply,

Example: I have n-year multitemporal images and an SHP file of points with an ID, and I would like to assign a class to each point for each year. This option could be used to perform independent validation of multitemporal maps or to collect samples for multitemporal classification.

Currently I use the plugin to collect samples independently of classifications, that is, without having the classification of the maps yet. But I have to fill the SHP attribute table by creating a field every year and fill manually, and then export it to Google Earth engine to validate or classify multitemporal images.

greetings.

**Comment by XavierCLL** [comment]
thanks for your explanation, I understand now, that is a good idea (and thanks for writing it here) but for very particular cases, something that we are planning (mid-term) is to have a sub-categories classification in the labelling buttons and with that, I think, cover your need in that case, is it? e.g. have every year in the main category and the classification in the sub-categories for each year ...

**Comment by efrainturpo** [comment]
Yes, I think that would be very good.
It would be good that in each window that is of a different date, also the option to choose the corresponding class. with options similar to https://github.com/lapig-ufg/tvi

**Reply by XavierCLL** [reply]
You can have that in Acatama right now, you can set up as many windows as you want (columns and rows in response design windows before opening it) and with the [GEE plugin](https://github.com/gee-community/qgis-earthengine-plugin) you can add any remote sensing image for specific date and load in each window in the Response design dialog.

**Reply by efrainturpo** [reply]
Of course I'm doing the same thing, but if you could have the option that in each window you can indicate the class that corresponds to the point, that would be great. I am currently filling out the shp attribute table manually, indicating which class the sample corresponds to in each year.

thank dear,

**Reply by efrainturpo** [reply]
In summary, my idea is that Acatama has an option of Temporary Visual Inspection. The visual part of Acatama is very good with GEE images, it would be great if it had the option to select or indicate the class in each window.

**Reply by XavierCLL** [reply]
Yes, I understand your point, thanks :+1:
